src/core/Link/index.tsx

```diff
@@ -21,8 +21,12 @@ export const Link = React.memo(
 
     const shouldOpenNewTab = external || props.to.search(httpRegExp) !== -1;
 
+    // NOTE: noreferrer의 사용은 noreferrer noopener을 함께 사용하는 것과 같은 동작을 하며:
```

Comment:
```suggestion
    // NOTE: noreferrer의 사용은 noreferrer noopener을 함께 사용하는 것과 같은 동작을 하며
```
(개취인 리뷰)
콜론 없는 게 더 깔끔하지 않을까여

Reply:
좋아여 수정하고 머지하겠음다